### PR TITLE
clarify definitions regarding continuity

### PR DESCRIPTION
**Pull request by sean-fitzpatrick**
Addresses #182

### CODE REVIEW

**Comment by APEXCalculus** [comment]
I *think* this is ok. I read stuff better once it is typeset, but I think it aligns with what the discussion was pointing to. I didn't merge as I'll let Alex weigh in, etc.

File: ptx/sec_limit_continuity.ptx [hunk 1]
```diff
@@ -314,18 +321,30 @@
...
+    But if <m>f</m> is undefined for <m>x\lt a</m>,
+    we can say that <m>f</m> is continuous at <m>a</m> without ambiguity.
   </p>
 
   <p>
-    We can make the appropriate adjustments to talk about continuity on half-open intervals such as
-    <m>[a,b)</m> or <m>(a,b]</m> if necessary.
+    Similarly, <xref ref="def_left_continuity">Item</xref> indcates that <m>f</m>
+    is <term>continuous from the left</term> at <m>b</m>,
+    unless <m>f</m> is not defined for <m>x\gt b</m>,
```
Comment (Alex-Jordan) [inline ptx/sec_limit_continuity.ptx:339]:
```suggestion
    and if <m>f</m> is not defined for <m>x\gt b</m>,
```

File: ptx/sec_limit_continuity.ptx [hunk 2]
```diff
@@ -314,18 +321,30 @@
...
+    in which case we can say <m>f</m> is continuous at <m>b</m>.
```
Comment (Alex-Jordan) [inline ptx/sec_limit_continuity.ptx:340]:
```suggestion
    we can simply say that <m>f</m> is continuous at <m>b</m>.
```

**Comment by Alex-Jordan** [comment]
I made two suggested edits (really one, covering two lines). I think this is a good balance between "technically and perfectly correct" and pedagogically helpful.

**Comment by sean-fitzpatrick** [comment]
Thanks. Those changes look good to me. @APEXCalculus if this is OK with you, it can be merged.

**Review by APEXCalculus — APPROVED** [review]

**Comment by Alex-Jordan** [comment]
My suggested edits didn't make it in, so I just committed a small edit
directly to main in GitHub. Just a heads up if anyone needs to pull
again.

On Wed, Jul 27, 2022 at 4:42 AM APEXCalculus ***@***.***> wrote:
>
> Merged #190 into main.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.Message ID: ***@***.***>



-- 
Alex Jordan
Mathematics Instructor
Portland Community College

**Comment by sean-fitzpatrick** [comment]
Ah, shoot. Sorry -- for some reason I didn't see the "commit change" button when I was reviewing on the 'Files changed' tab.

**Pull request merged**